sdk/sdk-common/src/common/implementation/TokenAmount.ts

```diff
@@ -23,4 +26,8 @@ export class TokenAmount implements Printable {
   public toString(): string {
     return `${this.amount} ${this.token.symbol}`
   }
+
+  public toWei(): string {
+    return new BigNumber(this.amount).times(this._weiFactor).toFixed(0)
+  }
```

Comment:
In fact `wei` refers to ether smallest unit, like `satoshi` for bitcoin. We can try to call it `baseUnit` however it was always confusing for me what base units means 😅  I guess we just need to agree how to call it.

Reply:
Yes, I agree. I could name it `toString()` and then pass the precision as a parameter, similar to what Ethers has. Not sure what is going to be more understandable

Reply:
lets ty `toBaseUnit()`